src/main/java/com/github/invictum/mei/CustomPlugin.java

```diff
@@ -0,0 +1,27 @@
+package com.github.invictum.mei;
+
+import org.bukkit.plugin.java.JavaPlugin;
+
+public abstract class CustomPlugin extends JavaPlugin {
+    private Backend backend = null;
+
+    public Backend getBackend() {
+        if (backend == null) {
+            this.reloadBackend();
+        }
+        return backend;
+    }
+
+    public void reloadBackend() {
+        String packageFolder = "com.github.invictum.mei.connectors.";
+        String backendType = getConfig().getString("backend", "MySql");
+
+        try {
+            this.backend = (Backend) Class.forName(packageFolder + backendType).newInstance();
```

Comment:
It will be better to selecting class by name, because this approach is case sensitive, it's better to refactor this method.